packages/core/ios/RNEmbrace/BundleID.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import EmbraceIO
+import CryptoKit
+
+private struct LastBundleComputation: Codable {
+    private static let STORAGE_KEY = "EMBRACE_LAST_BUNDLE_COMPUTATION"
+    var path: String
+    var computedAt: Date?
+    var id: String
+
+    func store () {
+        if let encoded = try? JSONEncoder().encode(self) {
+            UserDefaults.standard.set(encoded, forKey: LastBundleComputation.STORAGE_KEY)
+        }
+    }
+
+    static func fromStorage() -> LastBundleComputation {
+        if let storedData = UserDefaults.standard.object(forKey: LastBundleComputation.STORAGE_KEY) as? Data,
+           let loaded = try? JSONDecoder().decode(LastBundleComputation.self, from: storedData) {
+            return loaded
+        }
+
+        return LastBundleComputation(path: "", computedAt: nil, id: "")
+    }
+}
```

Comment:
I'd inject an "own" `UserDefaults` to prevent any problem in the future (like sombody migrating to another suite stuff, or things like that).
The only bad thing about injecting it (and having a instance variable with an own `userDefaults` is that you'll have to implement `Codable` manually.
So, i'd suggest something in the middle, like providing a utility to create an "Embrace" UserDefaults, like this:
```swift
extension UserDefaults {
    static var embrace: UserDefaults {
        UserDefaults(suiteName: "EmbraceReactNative") ?? .standard
    }
}
```
And then you can replace `store` and `fromStorage` using that utility:
```swift

func store () {
        if let encoded = try? JSONEncoder().encode(self) {
            UserDefaults.embrace.set(encoded, forKey: LastBundleComputation.STORAGE_KEY)
        }
    }

    static func fromStorage() -> LastBundleComputation {
        if let storedData = UserDefaults.embrace.object(forKey: LastBundleComputation.STORAGE_KEY) as? Data,
           let loaded = try? JSONDecoder().decode(LastBundleComputation.self, from: storedData) {
            return loaded
        }

        return LastBundleComputation(path: "", computedAt: nil, id: "")
    }
```

Reply:
makes sense, added